package.json

```diff
@@ -9,8 +9,8 @@
   "private": false,
   "scripts": {
     "prepublish": "yarn build:production",
-    "build:development": "NODE_ENV=development yarn run webpack --progress --mode development",
-    "build:production": "rm -rf ./dist/ && NODE_ENV=production env=production yarn run webpack",
+    "build:development": "yarn run webpack --env ${NODE_ENV:-development}",
+    "build:production": "rm -rf ./dist/ && yarn run webpack --env ${NODE_ENV:-production}",
```

Comment:
Hi!, here we forced NODE_ENV production because the task is named build:production, with that update now you can build a production version with a development env? Is what you want? Maybe you can try to make a generic build script with the node_env variable with that parameter :D

Reply:
Make sense, we could assume that when running the default build the environment is `prod`, merging the two tasks, something like this: 
`build: "rm -rf ./dist/ && yarn run webpack --env ${NODE_ENV:-production}"`.
What do you think?